internal/databases/greenplum/pg_hba_cfg_maker.go

```diff
@@ -93,3 +94,10 @@ func (m PgHbaMaker) primarySegments() []*cluster.SegConfig {
 	sort.Slice(primarySegments, func(i, j int) bool { return primarySegments[i].ContentID < primarySegments[j].ContentID })
 	return primarySegments
 }
+
+func (m PgHbaMaker) MaskedHostName(hostName string) string {
```

Comment:
Maybe no need to use a function to fix the host name when it is a ip address. The function name is a little confused.